enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -140,7 +140,7 @@ func flattenTrendsQuery(innerQuery string) string {
 	FROM (` + innerQuery + ")"
 }
 
-func addWhereClauses(q *query_builder.Query, tq *stpb.TrendQuery, reqCtx *ctxpb.RequestContext, lookbackWindowDays int32) error {
+func addWhereClauses(q *query_builder.Query, tq *stpb.TrendQuery, reqCtx *ctxpb.RequestContext, lookbackWindowDays int32, isExecutionQuery bool) error {
```

Comment:
I find the boolean here not super readable. Wondering if there are other ways to construct the code? for example, have two different methods for invocation and executions; extract common code in a separate method and etc.